lib/spack/spack/build_systems/autotools.py

```diff
@@ -8,15 +8,14 @@
 import fileinput
 import os
 import os.path
-import shutil
-import stat
 import sys
 import re
 from subprocess import PIPE
 from subprocess import check_call
 
 import llnl.util.tty as tty
-from llnl.util.filesystem import working_dir, force_remove
+from llnl.util.filesystem import working_dir, force_remove, copy, \
+    find, join_path
```

Comment:
Minor request, but can you:
```suggestion
import llnl.util.filesystem as fs
```
at least for the newly added functions and access them through the module object?

Reply:
fixed.